posts/2024-08-26-webassembly-targets-and-new-on-by-default-features.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: "WebAssembly targets and new on-by-default features"
```

Comment:
Discussion: should this use wasm terminology as in "proposals" or "WebAssembly proposals"? Or is target features less confusing?

Reply:
Maybe we should use the common terminology of "target features" so that it's more approachable.

"WebAssembly targets: change in default target-features"

Reply:
Yeah, I don't feel too strongly about which one, but I think we should stick to just one of them and not mix them.

Reply:
I've retitled the post, tried to be more consistent internally, and added a bit of discussion on proposal-vs-feature, but let me know if anything else needs to be improved!